test/driver.js

```diff
@@ -601,7 +604,7 @@ class Driver {
         task.pdfDoc.getPage(task.pageNum).then(
           page => {
             const viewport = page.getViewport({
-              scale: PixelsPerInch.PDF_TO_CSS_UNITS,
+              scale: PixelsPerInch.PDF_TO_CSS_UNITS * task.scale,
```

Comment:
Nit: To be consistent with e.g. the viewer usage, let's re-order this as `task.scale * PixelsPerInch.PDF_TO_CSS_UNITS` instead.